test/parallel/test-assert.js

```diff
@@ -130,7 +130,7 @@ assert.doesNotThrow(makeBlock(a.deepEqual, nb1, nb2));
 
 nameBuilder2.prototype = Object;
 nb2 = new nameBuilder2('Ryan', 'Dahl');
-assert.throws(makeBlock(a.deepEqual, nb1, nb2), a.AssertionError);
+assert.doesNotThrow(makeBlock(a.deepEqual, nb1, nb2), a.AssertionError);
```

Comment:
Minus `, a.AssertionError` and perhaps drop the `assert.doesNotThrow` call entirely?  The exception can just bubble up.